app/serializers/page_serializer.rb

```diff
@@ -32,6 +33,13 @@ def latest_blog_post_links
     end
   end
 
+  def related_links
+    object.related_links(3).map do |article|
+      article = article.mirrors.first if current_locale == 'cy'
+      build_json_link article
+    end
+  end
```

Comment:
This related links is the same behaviour as popular links. Should extracted to a method.